manifests/cf-manifest/operations.d/320-cc-set-loggregator-port.yml

```diff
@@ -0,0 +1,5 @@
+---
+# TODO: this value is changed in an opsfile from cf-deployment: aws.yml
+- type: replace
+  path: /instance_groups/name=api/jobs/name=cloud_controller_ng/properties/doppler?/port?
+  value: 443
```

Comment:
I wonder whether we would be better off not using the `aws.yml` opsfile as all it does is change this port for us to have to change it back. What do you think?

Reply:
We added it back because maybe one day they might add some stuff to `aws.yml` and we would miss it.

Reply:
Fair enough, that makes sense.